src/transformers/modeling_outputs.py

```diff
@@ -46,6 +46,25 @@ class BaseModelOutput(ModelOutput):
     attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithNoAttention(ModelOutput):
+    """
+    Base class for model's outputs, with potential hidden states.
+
+    Args:
+        last_hidden_state (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        hidden_states (`tuple(torch.FloatTensor)`, *optional*, returned when `output_hidden_states=True` is passed or when `config.output_hidden_states=True`):
+            Tuple of `torch.FloatTensor` (one for the output of the embeddings + one for the output of each layer) of
+            shape `(batch_size, num_channels, height, width)`.
```

Comment:
Some models, like SegFormer, don't have an initial embedding layer.

Reply:
Update: it's only PoolFormer

Reply:
> Maybe we can add a statement that says that the `hidden_states` optionally include the initial embeddings, if the model has an embedding layer.

+1 for @NielsRogge idea

Reply:
VAN as well